app/src/main/java/com/poupa/vinylmusicplayer/util/PreferenceUtil.java

```diff
@@ -74,6 +74,8 @@ public final class PreferenceUtil {
     private static final String ARTIST_GRID_SIZE = PrefKey.exportableKey("artist_grid_size");
     private static final String ARTIST_GRID_SIZE_LAND = PrefKey.exportableKey("artist_grid_size_land");
 
+    private static final String ALBUM_SHOW_FOOTER = PrefKey.exportableKey("album_show_footer")
```

Comment:
Build failed here, since missing ';'
I've allowed the CI builds on this PR
![image](https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/13333482/a04942ef-6808-4997-8098-4f1c8a0ea1fd)

You should be able to check on the build status

Reply:
Whoops, I missed that when trying to merge directly in GitHub. Thanks for catching that